bin/run-tests.js

```diff
@@ -84,10 +84,15 @@ function generateEachPackageTests() {
     .forEach(generateTestsFor);
 }
 
-function generateBuiltTests() {
+function generateBuiltTests(ie) {
```

Comment:
Instead of adding an arg, lets just push twice in the `testFunctions.push`'s below...